data_structures/linked_list/add_nodes.py

```diff
@@ -19,20 +20,40 @@ def __init__(self):
         self.head = None
```

Comment:
I think that it would be much more user friendly to call this class LinkedList instead of LL.